src/cascadia/ShellExtension/OpenTerminalHere.cpp

```diff
@@ -266,7 +266,16 @@ std::wstring OpenTerminalHere::_GetPathFromExplorer() const
         return path;
     }
 
-    auto shell = create_instance<IShellWindows>(CLSID_ShellWindows);
+    com_ptr<IShellWindows> shell;
+    try
+    {
+        shell = create_instance<IShellWindows>(CLSID_ShellWindows, CLSCTX_ALL);
+    }
+    catch (...)
+    {
+        //look like try_create_intance is not available no more

```

Comment:
```suggestion
        //look like try_create_instance is not available no more
```
Adding this as a suggestion to make your life a bit easier :)